cypress/Dockerfile

```diff
@@ -2,9 +2,17 @@ FROM cypress/included:5.2.0
 
 WORKDIR /app
 
-RUN npm install --save-dev cypress-axe@0.9.1 axe-core cypress-cucumber-preprocessor cosmiconfig
+RUN apt-get update && \
+  apt-get install -y python3-pip 
 
+RUN pip3 install boto3
+
+RUN npm install --save-dev cypress-axe@0.9.1 axe-core cypress-cucumber-preprocessor cosmiconfig cypress-tags
```

Comment:
cosmiconfig is not required here as far as I can tell; cypress-tags is a dependency of cypress-cucumber-preprocessor so we could remove that (as we're relying on the cypress-tags script wrapper which is in cypress-cucumber-preprocessor, not cypress-tags directly)

Reply:
yeah cosmiconfig was legacy from getting cypress-axe and cypress-cucumber to play together and that is true cypress-tags gets pulled in anyway